### PR TITLE
IA Vault key value pair

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @benghaziboy vault apparently needs a key, along with the stored token value

### CODE REVIEW

File: bulbs/content/tasks.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def post_article(content, body, fb_page_id, fb_api_url, fb_token_path, fb_dev_mo
 
 
 def delete_article(content, fb_api_url, fb_token_path):
-    fb_access_token = vault.read(fb_token_path)
+    fb_access_token = vault.read(fb_token_path)['authtoken']
```
Comment (benghaziboy) [inline bulbs/content/tasks.py]: is there a risk of this raising an error and will that block anything?

Comment (MichaelButkovic) [inline bulbs/content/tasks.py]: only if it's added to vault with the wrong key name

Comment (benghaziboy) [inline bulbs/content/tasks.py]: I'd prefer that it does a `.get(authtoken, None)` so we can check and not fail.

Comment (mparent61) [inline bulbs/content/tasks.py]: Best you can do is fail gracefully, by catching the error and raising exception to notify sentry.

Comment (MichaelButkovic) [inline bulbs/content/tasks.py]: updated

File: bulbs/content/models.py [hunk 2]
```diff
@@ -616,7 +616,7 @@ def delete_from_instant_article_api(sender, instance=None, **kwargs):
                     fb_token_path))
             return
 
-        fb_access_token = vault.read(fb_token_path)['authtoken']
+        fb_access_token = vault.read(fb_token_path).get('authtoken', None)
```
Comment (mparent61) [inline bulbs/content/models.py]: If you do this, then you need to check for `None` and raise. 
Also you don't need `, None`, this is the default and makes for cleaner code
```
     fb_access_token = vault.read(fb_token_path).get('authtoken')
```

**Comment by MichaelButkovic** [comment]
@mparent61 @benghaziboy updated

**Comment by mparent61** [comment]
👍

**Pull request merged**